website/docs/reference/commands/retry.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "About dbt retry command"
+sidebar_label: "retry"
+id: "retry"
+---
+
+`dbt retry` retries the last `dbt` command from the point of failure. If the previously executed `dbt` command was successful, `retry` will complete without compiling any nodes. 
+
+Retry works with the following commands:
+
+- [`build`](/reference/commands/build)
+- [`compile`](/reference/commands/compile)
+- [`seed`](/reference/commands/seed)
+- [`snapshot`](/reference/commands/build)
+- [`test`](/reference/commands/test)
+- [`run`](/reference/commands/run)
+- [`run-operation`](/reference/commands/run-operation)
+
+`dbt retry` will rerun from `error`, `failed`, `runtime error`, and `skipped` failures recorded in the run_results.json. If the command was successful with no errors, then it will rerun tests or freshness checks that warned. Executing `dbt retry` without correcting the previous failures will result in the same outcome. 
```

Comment:
Suggest moving this ^ paragraph up. I feel like it would flow nicely around line 8, because it feels connected to the 1st paragraph and would hate for it to be missed.